docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -84,22 +98,16 @@ Then, you can pull the image with this command:
 $ docker pull container-registry.oracle.com/middleware/soasuite:12.2.1.3
 ```
 
-Additional information about using this image is available in the
-[Oracle Container Registry](https://container-registry.oracle.com).
-
 #### Creating a SOA Suite Docker image
 
-If you prefer, you can also create a Docker image containing the SOA Suite binaries yourself.
-A [sample](https://github.com/oracle/docker-images/tree/master/OracleSOASuite)
-is provided in the Oracle GitHub account that demonstrates how to create a Docker image
-to run SOA Suite.  
+You can also create a Docker image containing the Oracle SOA Suite binaries. This is the recommended approach for the customers who have access to Oracle SOA and Oracle Service Bus bundle patches.
```

Comment:
This is the recommended approach for the customers who have access to Oracle SOA and Oracle Service Bus bundle patches. -> This is the recommended approach if you have access to Oracle SOA and Oracle Service Bus bundled patches.